fly.toml

```diff
@@ -10,6 +10,9 @@ processes = []
   cmd = "start.sh"
   entrypoint = "sh"
 
+[env]
+FLY_PRIMARY_REGION="ord" #for litestream
```

Comment:
We should probably set this to something that will fail-fast if they forget to set it explicitly...